components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -1535,6 +1536,9 @@ private void addServiceProviderOIDCProperties(Connection connection,
             addToBatchForOIDCPropertyAdd(processedClientId, spTenantId, prepStmtAddOIDCProperty,
                     SUBJECT_TYPE, consumerAppDO.getSubjectType());
 
+            addToBatchForOIDCPropertyAdd(processedClientId, spTenantId, prepStmtAddOIDCProperty,
```

Comment:
Can you point to the removal of previous persistence location

Reply:
earlier it was added as an sp property so no DAO level change was needed...removed adding it as an SP property